src/llfs/log_device_runtime_options.hpp

```diff
@@ -28,8 +28,9 @@ namespace llfs {
 struct LogDeviceRuntimeOptions {
   using Self = LogDeviceRuntimeOptions;
 
-  static constexpr usize kDefaultFlushDelayThreshold = 1 * kMiB;
+  static constexpr usize kDefaultFlushDelayThreshold = 128 * kKiB;
```

Comment:
Would we ever want this to be externally settable by a user? Have we tested other values and this is the best to use?